docs/TranslationGuide.md

```diff
@@ -29,18 +29,15 @@ following languages with the attached language code:
 ## Extracting translatable strings
 
 When adding new translatable strings to the GUI, they need to be added to each
-*LANG.json* file. This can be done with the following command:
+`*messages.<LANG>.xlf` file. This can be done with the following command:
```

Comment:
Should it really be "`*messages.<LANG>.xlf`"? Or rather "`messages.<LANG>.xlf`"?

Reply:
Fixed